DatadogSessionReplay/Sources/Recorder/PrivacyLevel.swift

```diff
@@ -8,6 +8,7 @@
 import DatadogInternal
 
 internal typealias PrivacyLevel = SessionReplayPrivacyLevel
+internal typealias TextAndInputPrivacyLevel = SessionReplayTextAndInputPrivacyLevel
```

Comment:
Why are we adding this privacy level but not the others?

Reply:
These two are typealiased because the belong to DatadogInternal (as they are shared with the WebViewTracking configuration (that's why they have extra SessionReplay prefix).

IMO worth reconsidering the whole setup once again.

maybe there's a way to keep all the models internal to SR.